fdbclient/ServerKnobs.cpp

```diff
@@ -633,6 +633,7 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( DURABILITY_LAG_REDUCTION_RATE,                      0.9999 );
 	init( DURABILITY_LAG_INCREASE_RATE,                        1.001 );
 	init( STORAGE_SERVER_LIST_FETCH_TIMEOUT,                    20.0 );
+	init( GET_SERVER_LIST_AND_PROCESS_CLASSES_TIMEOUT,         100.0 );
```

Comment:
This is probably too high. Since SS keeps 5s MVCC window, after 5s the SS will return an error. So this can be at most 5s.

Reply:
In a following up discussion with @bnamasivayam we decided to use 2s.